test/functional/rpc_whitelist.py

```diff
@@ -92,5 +94,56 @@ def run_test(self):
         self.log.info("Strange test 5")
         assert_equal(200, rpccall(self.nodes[0], self.strange_users[4], "getblockcount").status)
 
+        self.test_rpcwhitelistdefault_0_no_permissions()
+
+        # Replace file configurations
+        self.nodes[0].replace_in_config([("rpcwhitelistdefault=0", "rpcwhitelistdefault=1")])
+        with open(self.nodes[0].datadir_path / "bitcoin.conf", 'a', encoding='utf8') as f:
+            f.write("rpcwhitelist=__cookie__:getblockcount,getblockchaininfo,getmempoolinfo,stop\n")
+        self.restart_node(0)
+
+        # Test rpcwhitelistdefault=1
+        self.test_rpcwhitelistdefault_1_no_permissions()
+        self.test_rpcwhitelistdefault_1_with_user_permissions()
+
+    def test_rpcwhitelistdefault_0_no_permissions(self):
+        """
+        * rpcwhitelistdefault=0
+        * No Permissions defined
+        Expected result: * user3 (not whitelisted) can access any method
```

Comment:
In commit "test: Add test for rpcwhitelistdefault" (cfce5f5cbacf7d4c209112ac9d35856bed4a3c6b)

Note: since test was updated "user3" should now be "strangedude6"